src/builtins.c

```diff
@@ -1420,10 +1422,10 @@ size_t jl_static_show_x(JL_STREAM *out, jl_value_t *v, int depth)
         n += jl_printf(out, ".%s", jl_globalref_name(v)->name);
     }
     else if (jl_is_labelnode(v)) {
-        n += jl_printf(out, "%d:", jl_labelnode_label(v));
+        n += jl_printf(out, "%d:", (int)jl_labelnode_label(v));
```

Comment:
Same as above, better to have a format define for these, make them totally future proof, that is based on the type of what is being printed.